pkgs/tools/networking/tcptrace/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchurl, libpcap }:
+let
```

Comment:
I really dislike this indentation, which results in a lot of diff noise if an attribute gets changed.
also Why is most of this inside a let and not inside the mkDerivation itself? having it inside a let makes it harder to override with overrideAttrs imho